app/Jobs/RockTheVote/ImportRockTheVoteRecord.php

```diff
@@ -303,13 +309,17 @@ public function parseSmsSubscriptionTopicsChangeForUser(NorthstarUser $user)
         $fieldName = 'sms_subscription_topics';
         $currentSmsTopics = ! empty($user->{$fieldName}) ? $user->{$fieldName} : [];
 
+        info('User ' . print_r($user, true));
```

Comment:
We'll want to remove this before the update user SMS feature is enabled -- but this is where there is no `sms_subscription_topics` property on the user (`sms_status`, `email_subscription_status` and other expected fields are here)
```
[2020-04-17 19:33:49] local.INFO: User DoSomething\Gateway\Resources\NorthstarUser Object
(
    [attributes:protected] => Array
        (
            [id] => 5dd8d819fdce276a557e5895
            [_id] => 5dd8d819fdce276a557e5895
            [first_name] => Damaris
            [last_name] => O'Kon
            [last_initial] => O
            [photo] => 
            [email] => admin@dosomething.org
            [mobile] => *faker phone number*
            [facebook_id] => 5688
            [interests] => Array
                (
                )

            [birthdate] => 2008-08-07
            [addr_street1] => 
            [addr_street2] => 
            [addr_city] => 
            [addr_state] => 
            [addr_zip] => 
            [addr_source] => 
            [source] => factory
            [source_detail] => 
            [slack_id] => 
            [mobilecommons_id] => 
            [mobilecommons_status] => active
            [parse_installation_ids] => Array
                (
                )

            [email_subscription_status] => 1
            [email_subscription_topics] => 
            [causes] => Array
                (
                )

            [voter_registration_status] => registration_complete
            [voting_plan_status] => 
            [feature_flags] => Array
                (
                    [badges] => 
                    [refer-friends] => 1
                    [refer-friends-scholarship] => 1
                )

            [voting_plan_method_of_transport] => 
            [voting_plan_time_of_day] => 
            [voting_plan_attending_with] => 
            [language] => co
            [country] => NZ
            [drupal_id] => 
            [role] => admin
            [sms_status] => active
            [sms_paused] => 
            [last_accessed_at] => 2020-04-17T19:33:13+00:00
            [last_authenticated_at] => 2020-04-17T19:32:46+00:00
            [last_messaged_at] => 
            [updated_at] => 2020-04-17T19:33:13+00:00
            [created_at] => 2019-11-23T06:56:25+00:00
        )

    [dates:protected] => Array
        (
            [0] => created_at
            [1] => updated_at
        )

)
```

Reply:
ah, so just confirming this log is temporary for debugging?

Reply:
Yeah, I think it's fine to leave this in for now -- I'll add a TODO indicating the bug (hoping a set of fresh 👀after the weekend may help for finding why the heck this is happening)

Reply:
Amen to that 🙏A nice lil weekend will work wonders.

Reply:
@aaronschachter ya know, one thought: it _looks_ like we're still using an older version of Gateway. Could it be that we're still on the v1 Northstar API (via legacy gateway), and thus only seeing the [`Legacy/UserTransformer` attributes](https://github.com/DoSomething/northstar/blob/5e9dd80c38db209d70c0ae6281a22cf590e2fa0c/app/Http/Transformers/Legacy/UserTransformer.php) which _don't_ include the new fields?!

(Looks like we switched to /v2/ in Gateway 2.0 release! https://github.com/DoSomething/gateway/releases/tag/v2.0.0)

Reply:
(LOL this just popped into my head because I had a strikingly similar issue with the Northstar API over in Phoenix with fetching user data for Zendesk tickets...)

Reply:
argh, sure seems it! I was looking at gateway source code and forgot to check version in this app. If that's it, I'll open fix in new PR, thank you for the 🧠 💬 !